src/scenes/game/Chadburn.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { UnselectableImage } from "src/components/UnselectableImage";
-import styles from "./Game.module.css";
-import { GuessDial } from "./GuessDial";
+import { GuessDial } from "../../components/GuessDial";
```

Comment:
can you change this path to be relative to the project root? in this case, `src/components/GuessDial`

Reply:
Done!